<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStreamOffset</class>
 <widget class="QDialog" name="DialogStreamOffset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_VP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>19</y>
     <width>481</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>VP</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_KP">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>KP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_NP">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>260</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Message">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>290</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Stream will be offsetted to xxx hours
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_YES">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_NO">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Нет</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
